regen_go_proto_canonical_protos.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+bazel build //proto/cel/expr:all
+
+rm -vrf ./expr
+mkdir ./expr
+
+files=($(bazel aquery 'kind(proto, //proto/cel/expr:all)' | grep Outputs | grep "[.]pb[.]go" | grep "expr_go_proto" | sed 's/Outputs: \[//' | sed 's/\]//' | tr "," "\n"))
+for src in ${files[@]};
+do
+  $(cp $src ./expr)
```

Comment:
```suggestion
  cp -v "${src}" ./expr/
```

1. `$()` not needed
2. `"${src}"` escape best practice
3. `./expr/` ensures the destination is the dir (the last version of the script was copying each file to `./go` file, because the folder didn't exist and not having trailing `/` made shell interpret the destination as a file)